fossunited/foss_hackathon/doctype/foss_hackathon_participant/foss_hackathon_participant.py

```
@@ -27,7 +27,7 @@ class FOSSHackathonParticipant(Document):
         is_student: DF.Check
         localhost: DF.Link | None
         localhost_request_status: DF.Literal[
-            "Pending", "Pending Confirmation", "Accepted", "Rejected"  # noqa: F722, F821
+            "Pending", "Pending Confirmation", "Accepted", "Rejected"
```

Ruff (0.14.14)

[error] Undefined name `Pending`

(F821)